packages/devextreme/js/__internal/ui/html_editor/modules/m_resizing.ts

```diff
@@ -207,7 +207,12 @@ export default class ResizingModule extends BaseModule {
 
     if (option === 'enabled') {
       this.enabled = value;
-      value ? this._attachEvents() : this._detachEvents();
+      if (value) {
+        this._attachEvents();
+        this._createResizeFrame();
+      } else {
+        this.clean();
```

Comment:
[missed scenario]

Now the error can be presented if you try to repeatedly disable the resizing:
- you enable resizing
- set mediaResizing to { enabled: false } at runtime
- set mediaResizing.enabled to false at runtime

Reply:
[d3e2391](https://github.com/DevExpress/DevExtreme/pull/27707/commits/d3e2391b6a3fafa716b754490e326b0239065aaa)